openlibrary/macros/AvailabilityButton.html

```diff
@@ -0,0 +1,79 @@
+$def with(page, availability)
+
+$ waiting_loan = ctx.user and ctx.user.get_waiting_loan_for(page)
+$ user_loan = None
+$ my_turn_to_borrow = waiting_loan and waiting_loan['status'] == 'available' and waiting_loan['position'] == 1
+$ wlsize = availability.get('num_waitlist', 0)
```

Comment:
QUESTION: Same question as in the other PR. Is it misleading to say a failed waitlist number `get` should be treated as an empty waitlist?

Reply:
One problem is that it's inconsistent how its represented on IA's side. In most cases where there hasn't been a waitlist before, I believe the availability API reports `None` for the waitlist whereas if there was a waitlist and it has been decremented, the value shows as 0.

This being the case, I think the right answer is to keep the default as 0. This value is only used if there **is** a waitlist for the book. And if the waitlist value is somehow fundamentally wrong, that's a data problem.